npm/translatePolicy.go

```diff
@@ -221,6 +222,7 @@ func translateIngress(ns string, policyName string, targetSelector metav1.LabelS
 				util.IptablesSrcFlag,
 			)
 			entry.Specs = append(entry.Specs, targetSelectorIptEntrySpec...)
+			comment += "ALLOW-ALL-TO-" + targetSelectorComment + "-FROM-" + util.KubeAllNamespacesFlag
```

Comment:
I believe the max length for an iptables comment is 256, kubernetes objects have a name limit of 253, we should probably have some bounds checking when concatenating more strings that could be 253 chars long